front/app/api/admin_publications/types.ts

```diff
@@ -20,6 +20,7 @@ export interface IQueryParameters {
   removeNotAllowedParents?: boolean;
   onlyProjects?: boolean;
   filter_is_moderator_of?: boolean;
+  filter_user_is_moderator_of?: string;
```

Comment:
@IvaKop What is the different between `filter_is_moderator_of` and `filter_user_is_moderator_of`? :)

Reply:
Ok makes sense! Thanks! :)